vsts/pipelines/ci.yml

```diff
@@ -4,75 +4,91 @@ resources:
 variables:
 - group: Oryx
 
-jobs:
-- job: Job_Security
-  displayName: Security
-  condition: succeeded()
-  pool:
-    name: Hosted VS2017
-  steps:
-  - template: templates/_securityChecks.yml
+stages:
+  - stage: Build
+    displayName: Build and Test Oryx Images
+    jobs:
+    - job: Job_Security
+      displayName: Security
+      condition: succeeded()
+      pool:
+        name: Hosted VS2017
+      steps:
+      - template: templates/_securityChecks.yml
 
-- job: Job_SignBinaries
-  displayName: Build and Sign Oryx Binaries
-  pool:
-    name: VSEng-MicroBuildVS2017
-    demands:
-    - msbuild
-    - visualstudio
-  variables:
-    SignType: 'test'
-  steps:
-  - template: templates/_signbinary.yml
+    - job: Job_SignBinaries
+      displayName: Build and Sign Oryx Binaries
+      pool:
+        name: VSEng-MicroBuildVS2017
+        demands:
+        - msbuild
+        - visualstudio
+      variables:
+        SignType: 'test'
+        skipComponentGovernanceDetection: true
+      steps:
+      - template: templates/_signbinary.yml
 
-- job: Job_BuildImage
-  displayName: Build and Test Build Image
-  dependsOn: Job_SignBinaries
-  condition: succeeded()
-  pool:
-    name: OryxLinux
-  variables:
-    SignType: $[ dependencies.Job_SignBinaries.outputs['setSignTypeVariable.SignType'] ]  # map in the signtype variable
+    - job: Job_BuildImage
+      displayName: Build and Test Build Image
```

Comment:
There is something weird going on with these display names...can you investigate more?

![image](https://user-images.githubusercontent.com/6559784/59435982-9a55a300-8da3-11e9-8756-ce6137cce9ef.png)

Reply:
its concatenating job names with stage name when using step  template … i will address this to vsts community